src/d3a/models/myco_matcher/__init__.py

```diff
@@ -0,0 +1,53 @@
+from d3a.constants import FLOATING_POINT_TOLERANCE
+
+
+class MycoMatcher:
```

Comment:
Pay-as-clear is missing, there needs to be a second Myco class that deals with it

Reply:
Added!